pkg/goversion/go12.go

```diff
@@ -0,0 +1,21 @@
+// +build go1.12
```

Comment:
Will this filename change for each new compiler version? Or could we call it `minimum_compiler_version.go` or similar?